src/view/CLI.java

```diff
@@ -0,0 +1,68 @@
+package view;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+public class CLI {
+
+    InputStream is;
+    OutputStream os;
+
+    // default constructor with default IOstreanms (console IO):
+    public CLI() {
+        this.is = System.in;
+        this.os = System.out;
+    }
+
+
+    public CLI(String inputFileStr, String outputFileStr) throws IOException {
+
+        File initialFile = new File(inputFileStr);
+        this.is = new FileInputStream(initialFile);
+
+        File osFile = new File(outputFileStr);
+
+        // check if concat or rewrite the file:
+        boolean fileCreated = osFile.createNewFile();
+        if (!fileCreated) {
+            throw new IOException("Unable to create file at specified path. It already exists");
```

Comment:
Where this is caught? What will the user experience?

Reply:
The constructor coughing this, this message will be printed in the console. I have no idea how to implement this better.

Reply:
After the msg is printed what is the experience.. The program exits? Tries again?